docsrc/getting_started/jetpack.rst

```diff
@@ -63,17 +63,30 @@ Ensure libcusparseLt.so exists at /usr/local/cuda/lib64/:
     sudo cp -a libcusparse_lt-linux-sbsa-0.5.2.1-archive/lib/* /usr/local/cuda/lib64/
 
 
-Build torch_tensorrt
-~~~~~~~~~~~~~~
+Build torch_tensorrt wheel file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
This section should focus on source builds on device, I dont think we want to recommend the docker container as default